src/herder/TransactionQueue.h

```diff
@@ -59,13 +59,20 @@ class TransactionQueue
   public:
     static int64_t const FEE_MULTIPLIER;
 
-    enum class AddResult
+    enum class AddStatus
     {
         ADD_STATUS_PENDING = 0,
         ADD_STATUS_DUPLICATE,
         ADD_STATUS_ERROR,
         ADD_STATUS_TRY_AGAIN_LATER,
-        ADD_STATUS_COUNT
+        ADD_STATUS_COUNT,
+        ADD_STATUS_BAD_REPLACE_BY_FEE,
```

Comment:
this seems wrong: `ADD_STATUS_COUNT` is used as the marker for the upper bound for the enum values

Reply:
Good catch, thanks. It should be fixed now